README.md

```diff
@@ -254,6 +254,29 @@ try {
 catch (e) {
     console.log('share resource image to time line failed', e);
 }
+
+//Code example to share file use shareToSession API on android
+try {
+    var rootPath = fs.DocumentDirectoryPath;
+    var filePath = rootPath + '/email-signature-262x100.png'; // like
+
+    await fs.downloadFile('http://www.ncloud.hk/email-signature-262x100.png', filePath);
```

Comment:
> 这个PR请先关闭吧，我再重新提供示例代码

I'm -1 on this point at an open source project, because this pr is not active from the author @liuchungui, so we should guide him to finish this PR, not reject this one, thanks :-)